### PR TITLE
spellcheck readme

### PR DESCRIPTION
**Pull request by bwbohl**
* fix typos
* fix some sentences
* make the text more consistent in punctuation, hyphen-usage etc.

### CODE REVIEW

**Comment by rettinghaus** [comment]
Best thing about this PR that you misspelled spellcheck ;-)

File: README.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ In this document, you will learn how to contribute to the development of MEI by
 
 This repository contains all the source code of the MEI Schema and Guidelines:
 
- * [.github](.github): Configuration files for Github Actions workflows.
+ * [.github](.github): Configuration files for GitHub Actions workflows.
  * [customizations](customizations): TEI ODD files that allow you to build customized MEI schemata.
- * [source](source): Contains the source code, as expressed in TEI ODD. This includes the source code for the MEI Guidelines, and the MEI schema modules.
- * [submodules](submodules): A container for Git Submodules, that is third party developments that are needed for e.g. building this repository's contents, but not part of our codebase.
+ * [source](source): Contains the source code, as expressed in TEI ODD. This includes the source code for the MEI Guidelines and the MEI schema modules.
+ * [submodules](submodules): A container for Git Submodules, that is third-party developments that are needed, e.g., for building this repository's contents, but not part of our codebase.
```
Comment (ahankinson) [inline README.md]:
```suggestion
 * [submodules](submodules): A container for Git Submodules: third-party developments that are needed, e.g., for building this repository's contents, but are not part of our codebase.
```

**Comment by bwbohl** [comment]
@rettinghaus Fun fact: most of the time when I do spellchecking, I misspell the PR or commit message :-p

**Comment by bwbohl** [comment]
I just discovered that we're using different markdown syntax for the code blocks.

https://github.com/music-encoding/music-encoding/blob/163b671c61b63ae1e552c10908a1b2375b10f9b4/README.md?plain=1#L32-L34

Vs.

https://github.com/music-encoding/music-encoding/blob/163b671c61b63ae1e552c10908a1b2375b10f9b4/README.md?plain=1#L73-L75

I tried to get some info on this via GitHub, and it seems there are different options. `bash`, `console`, and more, as described in https://highlightjs.org/static/demo/

Do any of you know which performs best or best fits our purpose?

**Comment by ahankinson** [comment]
Looks godo to me.

**Review by ahankinson — APPROVED** [review]

**Review by ahankinson — APPROVED** [review]

**Comment by ahankinson** [comment]
I prefer the latter form since it lets you set the dialect of the code block (python, js, bash, etc.)

**Comment by bwbohl** [comment]
Me, too, but I was wondering which 'dialect' would be the most appropriate…

**Comment by ahankinson** [comment]
Oh... dunno. `bash` seems fine.

**Comment by rettinghaus** [comment]
GitHub uses [Linguist](https://github.com/github-linguist/linguist/). 
In their [configuration file](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml) you see that `bash` is an alias for `shell`. I opt for unified usage of the latter.

**Review by rettinghaus — APPROVED** [review]

**Pull request merged**